docs/index.adoc

```diff
@@ -437,3 +504,46 @@ creation of a new active wallet regardless of age.
 |Length of time a wallet has to fulfill a redemption.
 |Yes
 |`48 hours`
+
+|redemptionTxMaxFee
+|The max amount of satoshis per redemption that the wallet is allowed to pay miners.
+|Yes
+|`10000 satoshis`
+
+|redemptionTimeout
+|The amount of seconds a wallet has to fulfill a redemption request.
+|Yes
+|`172800 seconds = 48 hours`
+
+|redemptionTimeoutSlashingAmount
+|The amount of stake in T tokens that operators will lose if they do not fulfill a redemption request.
+|Yes
+|`10000 * 1e18` 
+
+|redemptionTimeoutNotifierRewardMultiplier 
+|The percentage of slashed stake from a redemption timeout that the notifier receives as a reward.
+|Yes
+|`100`
+
+4+s|Fraud 
+
+|fraudChallengeDepositAmount
+|The amount of ether a challenger must deposit in order to make a wallet prove honesty.
+|Yes
+|`2 ether`
+
+|fraudChallengeDefeatTimeout
+|The amount of time wallet has to wallet has to defend itself from a fraud challenge.
+|Yes
+|`7 days`
+
+|fraudSlashingAmount
+|The amount of T tokens that the operators of a wallet lose if they are defeated by a fraud challenge.
```

Comment:
I would prefer to not use the `defeated` here, as `defeated by a fraud challenge` can be confused with `defeating a fraud challenge`, which we use a lot.
Maybe better
```suggestion
|The amount of T tokens that the operators of a wallet lose if the fraud challenge does not get defeated.
```
?

Reply:
[54b7a3a](https://github.com/keep-network/tbtc-v2/pull/346/commits/54b7a3aa103d9dfe4e3cce4542eb7b703710ba88)